bulk-update/migration-tools/select.js

```diff
@@ -0,0 +1,79 @@
+import { select, selectAll } from 'unist-util-select';
+
+/**
+ * Retrieves the block information from a string and normalizes it.
+ * For example, the input `Block(option1, Option2)` returns
+ * `{ block: 'block', options: ['option1', 'option2'], variant: 'block (option1, option2)' }`
+ * And `block` returns `{ block: 'block', options: [], variant: 'block' }`
+ *
+ * @param {string} str - The input block string.
+ * @returns {Object} - An object containing the block name and options.
+ */
+export const getBlockInfo = (str) => {
+  if (!str) return null;
```

Comment:
I find this early return a bit odd since we should theoretically know what we are passing into the function at all times. 

I would expect more to find this pattern in the code if we are worried about a null:
```
const info = str ? blockInfo(str) : null 
```

Reply:
This was interesting, I found an empty table on a live page so I added this check.
I get what you're saying but as a framework we don't know how this function will be used in a future migration, perhaps setting a default empty string value is best here.
```js
export const getBlockInfo = (str = '') => {
```